core/domain/role_services.py

```diff
@@ -60,7 +60,7 @@
 ACTION_RATE_ANY_PUBLIC_EXPLORATION = 'RATE_ANY_PUBLIC_EXPLORATION'
 ACTION_SEND_MODERATOR_EMAILS = 'SEND_MODERATOR_EMAILS'
 ACTION_SUBSCRIBE_TO_USERS = 'SUBSCRIBE_TO_USERS'
-ACTION_SUGGEST_CHANGES_TO_EXPLORATION = 'SUGGEST_CHANGES_TO_EXPLORATION'
+ACTION_SUGGEST_CHANGES = 'SUGGEST_CHANGES'
```

Comment:
I think this is fine, we can keep a single role for making suggestions. So, you can either make suggestions for every type of entity or none.

Reply:
At the moment I think this is fine. I feel we should not have too many restrictions on who can create a suggestion (just logged in should be enough) and we should be more careful about who we allow to review (accept/reject) these suggestions.